src/main/resources/META-INF/plugin.xml

```diff
@@ -50,5 +51,10 @@
         <action id="fi.aalto.cs.apluscourses.intellij.actions.RequiredPluginsCheckerAction"
                 class="fi.aalto.cs.apluscourses.intellij.actions.RequiredPluginsCheckerAction"
                 text="Check Required Plugins"/>
+        <action id="Scala.RunConsole" class="fi.aalto.cs.apluscourses.intellij.actions.REPLAction"
+          overrides="true" text="Scala REPL..." description="Run Scala REPL">
+            <add-to-group group-id="ToolsMenu" anchor="last"/>
+            <keyboard-shortcut first-keystroke="control shift D" keymap="$default"/>
+        </action>
```

Comment:
almost identical to the [actual declaration of the action in Scala Plugin](https://github.com/JetBrains/intellij-scala/blob/69a0f0d5d4dd2c9a637e64761ac7d8bba49d2040/scala/scala-impl/resources/META-INF/scala-plugin-common.xml#L1477-L1482), the most important change is `overrides="true"` - this [seems](https://intellij-support.jetbrains.com/hc/en-us/community/posts/207623085-Intellij-plugin-development-how-to-prevent-an-action-from-occuring-like-closing-a-tab) to be the way to override actions (assume this works for IDE's own actions too as it works here)

Reply:
It would be ideal if we could kinda "decorate"

Reply:
and what if two plugins override the same action... :question:

Reply:
From discussion you [linked](https://intellij-support.jetbrains.com/hc/en-us/community/posts/207623085-Intellij-plugin-development-how-to-prevent-an-action-from-occuring-like-closing-a-tab):

> The idea is that ActionManager allows to get AnAction instance by its `id`. These `id` are registered in `.xml` files in `<actions>` block. If two plugins (or a plugin and platform) are trying to assign same `id` - it causes an error. Unless one of them passes `overrides` parameter.

So I assume/hope that if two plugins override the same action, it causes an error, which would be better than silently ignoring one